Production/python/TauL2Prod.py

```diff
@@ -154,8 +139,7 @@
     process = customizeHLTforMC(process)
 
 from TauMLTools.Production.myHLT_Train import update
-process = update(process,isData, options.outputFile)
-print options.outputFile
+process = update(process,isData, isData, options.outputFile)
```

Comment:
`applyGenTau` should be command line argument. We don't want to apply it for QCD MC